### ISSUE TITLE
feature: Jukebox mode (continuous random mod/xm)

### ISSUE THREAD

**Issue by defcon8** [steffest/BassoonTracker#51]
It would be great to have some sort of "Jukebox mode" where Bassoon tracker keeps playing random mod's and xm's. So I can let it play on my work during programming ;)

Great project!

**Comment by steffest** [comment]
Yep! That's will definitely happen soon (together with playlist support)

**Comment by blurymind** [comment]
I found myself yesterday, just hitting ranbom over and over and playing along with each one - trying to improvise some melody over it.
When you load a random mod, it doesnt play instantly, you have to manually play it, which kind of interrupts the flow

**Comment by steffest** [comment]
I think somewhere I impemented it already but decided to leave it out for now because in "jukebox mode" it's sometimes is hard to tell when a song is considered "finished" to jump to the next one.
I'll dig it out - thanks for reviving this issue, I kind of forgot about it.
